module-apps/application-settings/windows/network/NetworkWindow.cpp

```diff
@@ -136,14 +140,20 @@ namespace gui
             labelText += ": ";
             labelText += utils::translate("app_settings_network_volte_not_available");
         }
-        else if (volteState.beta) {
-            labelText += " <b>" + utils::translate("common_beta") + "</b>";
-        }
         labelText += "</text>";
 
         return labelText;
     }
 
+    auto NetworkWindow::getVolteSecondaryLabel() -> std::string
+    {
+        const auto &volteState = getVolteStateFromSettingsApp(application);
+        if (volteState.permitted && volteState.beta) {
+            return "<text><b>" + utils::translate("common_experimental") + "</b><text>";
+        }
+        return "";
```

Comment:
You can use `{}` instead of `""`. This should be more efficient ;)

Reply:
Changed! I didn't know that to be honest, I thought they're equivalent :open_mouth: